### PR TITLE
btn-lg is smaller than input-lg

### PR DESCRIPTION
**Pull request by fassetar**
In Chrome, Safari, and Firefox (not IE10+) btn-lg is a pixel off and should be 46px but instead its 45px. Here is [some code](http://plnkr.co/edit/zS4bT68QREkpOTaImqEX?p=preview) that I'm working with. What I've done in my projects is just add a height:46px to btn-lg but I think a better solution is to just change the @line-height-large to 1.34 instead of 1.33.

### CODE REVIEW

**Comment by twbs-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 69d1aa7a9a4f9572344de6d724dd4af1bcd0ff24
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/45714198

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by twbs-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: ec7bfd698fb551861da34c465204576b9bf3651f
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/45717175

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by twbs-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 2c2a8d63362fe5a271856c6a4811b6d2828eb4ba
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/46883953

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by twbs-savage** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 73c1f2b7aa143f6c19664e801c8fbdb276921512
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/46885995

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: js/tests/index.html [hunk 1]
```diff
@@ -4,12 +4,11 @@
     <meta charset="utf-8">
     <title>Bootstrap Plugin Test Suite</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
     <!-- jQuery -->
     <script src="vendor/jquery.min.js"></script>
 
     <!-- QUnit -->
-    <link rel="stylesheet" href="vendor/qunit.css" media="screen">
+    <link rel="stylesheet" href="vendor/qunit.css" media="screen">
```
Comment (mdo) [inline js/tests/index.html]: Trailing whitespace here

**Comment by mdo** [comment]
I really don't want to set the height of buttons, which is why I haven't pursued this much. I'll check in on the problem again when I can.

**Comment by fassetar** [comment]
I agree, I will play around some more but here are my [notes](http://anthonyfassett.blogspot.com/2015/01/bootstrap-btn-lg-is-smaller-than-input.html).

**Comment by mdo** [comment]
Closing as we aren't going to be setting the height.

**Pull request closed without merging**